src/unix/timer.c

```diff
@@ -71,7 +72,7 @@ int uv_timer_start(uv_timer_t* handle,
   if (uv__is_active(handle))
     uv_timer_stop(handle);
 
-  clamped_timeout = handle->loop->time + timeout;
+  clamped_timeout = (handle->timeout <= 0 ? handle->loop->time : handle->timeout) + timeout;
```

Comment:
I meant the repeating timer is not running on the EXACT fixed points due to inaccurate loop->time.
It is updated automatically in uv loops and I don't have to run uv_update_time to update it in repeating timers.